spec/system/nav_bar_system_spec.rb

```diff
@@ -20,7 +19,7 @@
     login_as_user(:tux)
 
     within(page.find('pzsh-menu')) do
-      expect(page).to have_text('Admin')
+      expect(page.body.downcase).to have_text('admin')
```

Comment:
Isn't it possible anymore to check the old way (without the downcase and check for "Admin" and not "admin")